prep_release.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+""" This script builds and runs automated tests for a new moldesign release.
+It's manual for now, will be replaced by Travis or Jenkins soon.
+
+You shouldn't run this unless you know exactly what you're doing and why you're doing it
```

Comment:
😄 
(actually, that's leftover from the `moldesign` version of this script, which can do bad things to your docker registry and git tags if you do it wrong)